sql-prepared-plan-cache.md

```diff
@@ -17,6 +17,78 @@ When the execution plan cache is enabled, in the first execution every `Prepare`
 
 TiDB also supports execution plan caching for some non-`PREPARE` statements, similar to the `Prepare`/`Execute` statements. For more details, refer to [Non-prepared plan cache](/sql-non-prepared-plan-cache.md).
 
+## Session-level and instance-level Prepared Plan Cache
+
+TiDB supports both session-level plan cache and instance-level plan cache. Session-level plan cache maintains a separate cache for each session, while instance-level plan cache maintains a single shared cache across all sessions in an instance.
+
+To enable or disable instance-level plan cache, you can use the [`tidb_enable_instance_plan_cache`](/system-variables.md#tidb_enable_instance_plan_cache-new-in-v840) system variable. When you set this variable to `ON`, instance-level plan cache is used. Otherwise, session-level plan cache is used.
+
+When you set `tidb_enable_instance_plan_cache` to `ON`, plans can be shared across different sessions. The following is an example:
+
+```sql
+-- Execute the following SQL statements in session 1.
+mysql> PREPARE st FROM "SELECT a FROM t WHERE a<?";
+Query OK, 0 rows affected (0.00 sec)
+
+mysql> SET @a=1;
+Query OK, 0 rows affected (0.00 sec)
+
+mysql> EXECUTE st USING @a;
+Empty set (0.01 sec)
+
+-- Execute the following SQL statements in session 2.
+mysql> PREPARE st FROM "SELECT a FROM t WHERE a<?";
+Query OK, 0 rows affected (0.00 sec)
+
+mysql> SET @a=30;
+Query OK, 0 rows affected (0.00 sec)
+
+mysql> EXECUTE st USING @a;
+Empty set (0.01 sec)
+
+mysql> SELECT @@last_plan_from_cache;
++------------------------+
+| @@last_plan_from_cache |
++------------------------+
+|                      1 |
++------------------------+
+```
+
+Starting from v8.5.0, TiDB provides two system tables to view entries in the instance-level plan cache:
+
+* `information_schema.tidb_plan_cache`: shows entries of instance plan cache in the current TiDB instance.
+* `information_schema.cluster_tidb_plan_cache`: shows entries of instance plan cache across all available TiDB instances.
```

Comment:
Shall we include a page to describe the view under https://docs.pingcap.com/zh/tidb/stable/information-schema/ ?